src/AElf.WebApp.MessageQueue/SendMessage.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using AElf.WebApp.MessageQueue.Enum;
+using AElf.WebApp.MessageQueue.Provider;
+using AElf.WebApp.MessageQueue.Services;
+using Microsoft.Extensions.DependencyInjection;
+using Volo.Abp.BackgroundWorkers;
+
+namespace AElf.WebApp.MessageQueue;
+
+public class SendMessage
```

Comment:
Change to SendMessageService and add ISendMessageService interface. Register it in module